glowing-waffles/notebooks/source_detection.py

```diff
@@ -0,0 +1,11 @@
+from ccdproc import CCDData
+from astropy.stats import sigma_clipped_stats
+from photutils import daofind
+
+ccd = CCDData.read('/Users/Nathan/Desktop/Astro-Project/Testing/M52-002R.fit', unit="adu")
```

Comment:
Could you please remove this line? Otherwise it will get run every time gw is used...